asyncapi-cli/src/main/java/io/ballerina/asyncapi/websocketscore/generators/asyncspec/service/AsyncApiRemoteMapper.java

```diff
@@ -91,6 +108,42 @@ public class AsyncApiRemoteMapper {
         this.semanticModel = semanticModel;
     }
 
+    public static boolean containsCloseFrameSchema(AsyncApi25ComponentsImpl components) {
+        if (components == null || components.getSchemas() == null) {
```

Comment:
Let's use this approach when we are checking nullability. Pls check for other places as well.
```suggestion
        if (Objects.isNull(components) || Objects.isNull(components.getSchemas())) {
```